include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -7,7 +7,7 @@
 #include <string.h>
 
 // This value is used in structures passed to ORT so that a newer version of ORT will still work with them
-#define ORT_API_VERSION 7
+#define ORT_API_VERSION 8
```

Comment:
API version needs bumping because only starting this "in development" API version (which is the same as the custom op version), do we support defining inputs/outputs as optional